inst/stan/epinowcast.stan

```diff
@@ -86,14 +86,14 @@ parameters {
 transformed parameters{
   vector[refp_fnrow] refp_mean;
   vector[refp_fnrow] refp_sd;
-  matrix[dmax, refp_fnrow] pmfs; // sparse report distributions
```

Comment:
I threw out the pmfs and did not replace it with the hazards - could be changed if having the hazards as a transformed parameter is important.

Reply:
No don't think, so this is sensible.